### ISSUE TITLE
Glossary definitions don't play nice with obsolete terms

### ISSUE THREAD

**Issue by Villawhatever** [Fryyyyy/Fryatog#136]
```
<DoubleFried> !define bury
<DoubleFried> ah no Glossary for Fryatog :(
<Volo> how dare you. !define bury (obsolete)
<Fryatog> Volo: Bury (Obsolete): A term that meant "put [a permanent] into its owner's graveyard." In general, cards that were printed with the term "bury" have received errata in the Oracle card reference to read, "Destroy [a permanent]. It can't be regenerated," or "Sacrifice [a permanent]."
```

**Comment by Fryyyyy** [comment]
1) Can special case Bury
2) Can check the prefix (strings.hasPrefix)
3) Strip () from glossary names

**Comment by Fryyyyy** [comment]
I believe this was fixed in d7e3342188be3788bb3262abe1c263bfe86bbf34
